src/cljs/xt_play/app.cljs

```diff
@@ -17,26 +18,34 @@
 
 ;; 'my.app.thing :trace  ;; Some namespaces you might want detailed logging
 
+(defn- decompress-params [{:keys [uri-version] :as query-params}]
+  (let [decompress (case uri-version
```

Comment:
It could be more intuitive to be an `if` if there's only one case with a default (I know, I'm a hypocrite)

Reply:
I left it like this because I’ve changed the uri format enough times to know I’ll change it again 😅